src/seedu/addressbook/ui/Formatter.java

```diff
@@ -0,0 +1,52 @@
+package seedu.addressbook.ui;
+
+import java.util.List;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+/**
+ * Formatter will have public methods used by TextUI to format text.
+ * @author unusep
```

Comment:
You can omit the `@author` tag as we expect multiple developers to edit the code.